var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -50,14 +50,14 @@ class Abinit(Package):
     # Versions before 8.0.8b are not supported.
     version("8.0.8b", "abc9e303bfa7f9f43f95598f87d84d5d")
 
-    variant('mpi', default=True,
-            description='Builds with MPI support. Requires MPI2+')
-    variant('openmp', default=False,
-            description='Enables OpenMP threads. Use threaded FFTW3')
-    variant('scalapack', default=False,
-            description='Enables scalapack support. Requires MPI')
+    variant("mpi", default=True,
```

Comment:
i saw a mixture of `'` and `"` in Spack. We should probably use one of another consistently.